tools/docker/ubuntu_2204.dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update && apt-get install -y gnupg2 --no-install-recommends curl &&
     curl -fsSL http://repo.radeon.com/rocm/rocm.gpg.key | gpg --dearmor -o /etc/apt/trusted.gpg.d/rocm-keyring.gpg
 
 # Add rocm repository
-RUN sh -c "echo 'deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/rocm-keyring.gpg] http://repo.radeon.com/rocm/apt/5.5 jammy main' > /etc/apt/sources.list.d/rocm.list"
+RUN sh -c "echo 'deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/rocm-keyring.gpg] http://repo.radeon.com/rocm/apt/5.7 jammy main' > /etc/apt/sources.list.d/rocm.list"
```

Comment:
I recently tried this docker file, it didn't work out of the box fo rme. Would be good to check it works as expected.  As a separate change list/PR though. For now this is fine.  There is this one https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/2017

Reply:
Oh you mean pull it out of this PR? I've update doc preqs since we're using sphinx now and mirrored what we did from our regular 20.04 Dockerfile.